roles/openshift_logging/files/generate-jks.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/sh
+
+set -ex
+
+keytool -genkey -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -keypass kspass -storepass kspass \
+-keyalg RSA -keysize 2048 -validity 712 -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+keytool -certreq -alias {{component}} -keystore {{generated_certs_dir}}/{{component}}.jks -storepass kspass \
+-file {{generated_certs_dir}}/{{component}}-jks.csr -keyalg RSA -dname "CN={{component}}, OU=OpenShift, O=Logging" {{extensions.stdout}}
+
+# this currently fails since we don't provide openssl as part of the ES image...
+openssl ca -in {{generated_certs_dir}}/{{component}}-jks.csr -notext -out {{generated_certs_dir}}/{{component}}-jks.crt \
+-config {{generated_certs_dir}}/signing.conf -extensions v3_req -batch -extensions server_ext
```

Comment:
@ewolinetz ah, thanks for testing that out. We do have another role that uses openssl directly, but it's limited to etcd currently and I'm not sure we want to extend it's use, since we only use it because the openshift ca doesn't support combined client/server certs.